### PR TITLE
Added a staff section to the about page.

### PR DESCRIPTION
**Pull request by karthik**
Can you a bit more work and update on pictures/css but something we could have for now. Someone else please merge.

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -31,7 +31,100 @@
 
     <div class="container">
         <div class="row">
-            <section class="crew-3 col-sm-8" id="leadership">
+
+<!-- Begin Staff -->
+<section class="crew-3 col-sm-8" id="leadership">
```
Comment (sckott) [inline about/index.html]: to  `id="staff"` i think

**Comment by sckott** [comment]
and rsync, or not yet?

File: about/index.html (hunk 1 above)
Comment (karthik) [inline about/index.html]: fixed, nice catch.

**Comment by karthik** [comment]
Yes, and please rsync.

**Comment by sckott** [comment]
done

**Pull request merged**